JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -513,6 +513,142 @@ class AppInsightsTests extends TestClass {
         });
 
         this.testCase({
+            name: "AppInsightsTests: set authID context is applied",
```

Comment:
Can we take advantage of sinon and only verify that `AppInsights` methods call the respective `UserContext` methods, which are tested separately?